.github/workflows/publish.yml

```diff
@@ -3,28 +3,27 @@ name: publish_docs
 on:
   push:
     branches:
-    - master
+      - undefined # to prevent it from firing
```

Comment:
I think it can be `branches: []` (not tested)

Reply:
Once it is in master, I'll experiment and see if [] would work. So leaving as it is for now.